ts/sagas/notifications.ts

```diff
@@ -51,16 +55,20 @@ export function* updateInstallationSaga(
         }
       }
     );
-
     /**
      * If the response isLeft (got an error) dispatch a failure action
      */
     if (response.isLeft()) {
       throw Error(readableReport(response.value));
     }
-    return response.value.status;
+    if (response.value.status === 200) {
+      yield put(
+        notificationsInstallationTokenRegistered(
```

Comment:
Atm we track only `updateNotificationInstallationFailure`. Do you think could be useful to track also this action, for troubleshooting purpose?
Or maybe could be useful to track also the "not required update because the token is the same". What you think?

Reply:
this is a good point for analytics
I guess tracking both could be a good choice.
https://github.com/pagopa/io-app/pull/3295/commits/92c6f38ccb397b4f501bbc2ef60e9d40d7e4d841

Reply:
Could be useful to track also when the token is registered with success?

Reply:
`notificationsInstallationTokenRegistered` represents that event: backend saved the notification token sent by the app
It is also tracked in the analytics middleware
Are you talking about to add another specific action?

Reply:
Ops, didn't see in analytics!